one-ui/src/main/java/com/marklogic/hub/curation/services/FlowManagerService.java

```diff
@@ -365,4 +370,13 @@ public Flow stop(String flowName) {
         return getFlow(flowName);
     }
 
+    protected FlowRunnerImpl getFlowRunner() {
+        lastFlowRunner = new FlowRunnerImpl(this.hubConfig);
+        return lastFlowRunner;
+    }
+
+    // TODO public so the FlowControllerTest can see it
+    public FlowRunnerImpl getLastFlowRunner() {
```

Comment:
FlowManagerService isn't accomplishing anything in terms of reuse - FlowController is its only client (just like in QS). Its name doesn't fit with our naming conventions either. How about just shoving all of it into FlowController, and then you can just do the trick I mentioned above for getting a reference to lastFlowRunner?